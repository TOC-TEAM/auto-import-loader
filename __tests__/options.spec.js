import webpack from './helpers/compiler'

describe('when applied with option', () => {
  it('matches snapshot for `entry` and `asset`', async () => {
    const config = {
      loader: {
        test: /.jsx?$/,
        options: { entry: 'src/pages', asset: 'src/assets/js' },
      },
    }

    const stats = await webpack('src/pages/test/index.js', config)
    const [module] = stats.toJson().modules
    expect(module.source).toMatchSnapshot()
  })

  it('matches snapshot for `cssPreprocessor`', async () => {
    const config = {
      loader: {
        test: /.jsx?$/,
        options: {
          entry: 'src/pages',
          asset: 'src/assets/js',
          cssPreprocessor: 'stylus',
        },
      },
    }

    const stats = await webpack('src/pages/test/index.js', config)
    const [module] = stats.toJson().modules
    expect(module.source).toMatchSnapshot()
  })
})
