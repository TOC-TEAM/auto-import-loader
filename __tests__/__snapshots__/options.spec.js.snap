// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when applied with option matches snapshot for \`cssPreprocessor\` 1`] = `""`;

exports[`when applied with option matches snapshot for \`entry\` and \`asset\` 1`] = `""`;
